modules/system/lang/ar/lang.php

```diff
@@ -48,4 +48,8 @@
     'settings' => [
         'menu_label' => 'الإعدادات',
     ],
+	'pagination' => [
```

Comment:
@mabehiry Can you change the tab character to spaces please? :)

Reply:
Ok, I did.
Thank you